crates/iceberg-catalog/Cargo.toml

```diff
@@ -109,4 +112,5 @@ tower = { workspace = true }
 tracing-subscriber = { workspace = true }
 tokio = { workspace = true, features = ["test-util"] }
 tracing-test = "0.2.5"
-
+hdfs-native = { workspace = true, features = ["integration-test"] }
+serial_test = { version = "3.2.0", features = ["async"] }
```

Comment:
needed if we wanna test against minidfs but minidfs is incompatible with nextest since it spawns a java hadoop cluster with hardcoded ports etc so concurrency means boom